docs/deployment.md

```diff
@@ -124,7 +132,7 @@ EXPOSE 3000
 # Uncomment the following line in case you want to disable telemetry.
 # RUN npx next telemetry disable
 
-CMD ["node_modules/.bin/next", "start"]
+CMD ["yarn", "start"]
```

Comment:
I changed this because Google Cloud Run requires you to forward a port to the start script. Then I realized it's probably confusing if `package.json` isn't the source of truth for how you're starting your application, so it's now looking there.